src/scss/utilities/_borders.scss

```diff
@@ -8,3 +8,7 @@
     border-color: $value;
   }
 }
+
```

Comment:
I don't think you need to do this. Bootstrap has a bunch of utils for this kind of thing: https://getbootstrap.com/docs/4.2/utilities/borders/